gradle.properties

```diff
@@ -21,15 +21,15 @@ shadowGradlePlugin=2.0.3
 slf4jApiVersion=1.7.25
 
 # Version and base tags can be overridden at build time.
-connectorVersion=0.6.1-SNAPSHOT
-pravegaVersion=0.6.0
+connectorVersion=0.6.1
+pravegaVersion=0.6.1
 
 # flag to indicate if Pravega sub-module should be used instead of the version defined in 'pravegaVersion'
 usePravegaVersionSubModule=false
 
 # These properties are only needed for publishing to maven central
 # Pravega Signing Key
-signing.keyId=05949AF6
+signing.keyId=F56ED3A7
```

Comment:
should we put signing key id in gradle properties?
we do not include it in pravega

Reply:
Good idea to remove that.